modules/cnv-vm-wizard-fields-web.adoc

```diff
@@ -45,16 +45,13 @@ endif::[]
 |Cloned Disk
 |Provision source is a cloned disk.
 
-|Import
-|Import virtual machine from a supported provider.
-
 |Operating System
 |
-|A list of operating systems available in the cluster. This is the primary operating system for the virtual machine. If you select `Import` as the *Provider Source*, the operating system is filled in automatically, based on the operating system of the VMware virtual machine being imported.
+|A list of operating systems available in the cluster. This is the primary operating system for the virtual machine.
 
 |Flavor
 |small, medium, large, tiny, Custom
-|Presets that determine the amount of CPU and memory allocated to the virtual machine.
+|Presets that determine the amount of CPU and memory allocated to the virtual machine. The presets displayed for Flavor are determined by the operating system and Workload profile.
```

Comment:
Drop workload profile here because it comes after the Flavor. It's confusing to say a value is provided by an option that is selected afterwards. This will be fixed in the next release I believe.